src/typings.d.ts

```diff
@@ -0,0 +1 @@
+declare module "@hermeznetwork*";
```

Comment:
To make typescript shut up about the untyped `hermezjs` library.

Adding the types we use from the library is the main task of the next PR.

Reply:
Mmm interesting, didn't know about this ambient modules! I thought that just setting the `skipLibCheck: true` in the `tsconfig.json` was enough to prevent complaining about untyped modules.